lib/rules/require-space-before-semicolon.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Requires space before semicolon.
+ *
+ * Types: `Boolean`
+ *
+ * Values: `true` to require a single space before any semicolon.
+ *
+ * #### Example
+ *
+ * ```js
+ * "requireSpaceBeforeSemicolon": true
```

Comment:
I wonder who would want that rule, yeah, we usually ask for the opposite rule, but i think we should be more soft on that.